### PR TITLE
update win32-and-centennial-app-policy-configuration.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #960 ,
added note about parameters that cannot be changed via OMA-URI

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/win32-and-centennial-app-policy-configuration.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ When the ADMX policies are imported, the registry keys to which each policy is w
 > [!Warning]
 > Some operating system components have built in functionality to check devices for domain membership. MDM enforces the configured policy values only if the devices are domain joined, otherwise it does not. However, you can still import ADMX files and set ADMX-backed policies regardless of whether the device is domain joined or non-domain joined.
 
+> [!NOTE]
+> Settings, that cannot be configured using custom policy ingestion, have to be set by pushing the appropriate registry keys directly (for example, by using PowerShell script).
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/win32-and-centennial-app-policy-configuration.md]:
```suggestion
> Settings that cannot be configured using custom policy ingestion have to be set by pushing the appropriate registry keys directly (for example, by using PowerShell script). 
```

**Comment by mypil** [comment]
@MaratMussabekov  - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were successfully applied, thank you!

**Pull request merged**